src/main/fc/fc_core.c

```diff
@@ -434,6 +496,66 @@ void processRx(timeUs_t currentTimeUs)
     }
 
+    // If runaway_takeoff_prevention is enabled, accumulate the amount of time that throttle
+    // is above runaway_takeoff_deactivate_throttle with the any of the R/P/Y sticks deflected
+    // to at least runaway_takeoff_stick_percent percent while the pidSum on all axis is kept low.
+    // Once the amount of accumulated time exceeds runaway_takeoff_deactivate_delay then disable
+    // prevention for the remainder of the battery.
+
+    if (ARMING_FLAG(ARMED)
+        && pidConfig()->runaway_takeoff_prevention
+        && !runawayTakeoffCheckDisabled
+        && !STATE(FIXED_WING)) {
+
+        // Determine if we're in "flight"
+        //   - motors running
+        //   - throttle over runaway_takeoff_deactivate_throttle_percent
+        //   - sticks are active and have deflection greater than runaway_takeoff_deactivate_stick_percent
+        //   - pidSum on all axis is less then runaway_takeoff_deactivate_pidlimit
+        inFlight = false;
+        if (!feature(FEATURE_MOTOR_STOP) || isAirmodeActive() || (calculateThrottleStatus() != THROTTLE_LOW)) { // are motors running?
+            float pidSumLimit = pidConfig()->runaway_takeoff_deactivate_pidlimit * 10.0f;
+            if ((calculateThrottlePercent() >= pidConfig()->runaway_takeoff_deactivate_throttle)
+                && areSticksActive(pidConfig()->runaway_takeoff_deactivate_stick)
+                && (fabsf(axisPID_P[FD_PITCH] + axisPID_I[FD_PITCH] + axisPID_D[FD_PITCH]) < pidSumLimit)
+                && (fabsf(axisPID_P[FD_ROLL] + axisPID_I[FD_ROLL] + axisPID_D[FD_ROLL]) < pidSumLimit)
+                && (fabsf(axisPID_P[FD_YAW] + axisPID_I[FD_YAW]) < pidSumLimit)) {
+                
+                inFlight = true;
+                if (runawayTakeoffDeactivateUs == 0) {
+                    runawayTakeoffDeactivateUs = currentTimeUs;
+                }
+            }
+        }
+
+        // If we're in flight, then accumulate the time and deactivate once it exceeds runaway_takeoff_deactivate_delay milliseconds
+        if (inFlight) {
+            if (runawayTakeoffDeactivateUs == 0) {
+                runawayTakeoffDeactivateUs = currentTimeUs;
+            }
+            if ((cmpTimeUs(currentTimeUs, runawayTakeoffDeactivateUs) + runawayTakeoffAccumulatedUs) > (pidConfig()->runaway_takeoff_deactivate_delay * 1000)) {
+                runawayTakeoffCheckDisabled = true;
+            }
+
+        } else {
+            if (runawayTakeoffDeactivateUs != 0) {
+                runawayTakeoffAccumulatedUs += cmpTimeUs(currentTimeUs, runawayTakeoffDeactivateUs);
+            }
+            runawayTakeoffDeactivateUs = 0;
+        }
+        if (runawayTakeoffDeactivateUs == 0) {
+            DEBUG_SET(DEBUG_RUNAWAY_TAKEOFF, 2, 0);
+            DEBUG_SET(DEBUG_RUNAWAY_TAKEOFF, 3, runawayTakeoffAccumulatedUs / 1000); // Accumulated in-flight time in ms
+        } else {
+            DEBUG_SET(DEBUG_RUNAWAY_TAKEOFF, 2, 1);  // deactivation countdown is active
```

Comment:
Better use defines instead of `0`, `1` here. This will also eliminate the need for the comment.

The same goes for the debug index, with defines it will be immediately clear to the reader what the logged values are for.

Reply:
Done.